lib/components/user/monitored-trip/saved-trip-screen.js

```diff
@@ -148,8 +147,8 @@ class SavedTripScreen extends Component {
 
       const clonedSchemaShape = clone(validationSchemaShape)
       clonedSchemaShape.tripName = yup.string()
-        .required('Please enter a trip name')
-        .notOneOf(otherTripNames, 'Another saved trip already uses this name. Please choose a different name.')
+        .required('components.SavedTripScreen.tripNameRequired')
+        .notOneOf(otherTripNames, 'components.SavedTripScreen.tripNameAlreadyUsed')
```

Comment:
Please do not pass message IDs like this to yup validation schemas. This results in message IDs not being hard-coded within a `intl.format` call or `FormattedMessage` component and is therefore hard to automatically keep track of which IDs are in the code.

Instead, please omit the message from the yup validation schema:

```js
tripName: yup.string().required().notOneOf(otherTripNames)
```

Reply:
After seeing this in further action, I think you could use the message as an identifier to see what error occurred, but that identifier shouldn't be a message ID to avoid confusion.